app/models/client.rb

```diff
@@ -0,0 +1,49 @@
+class Client < ActiveRecord::Base
+  has_many :payload
```

Comment:
a `Client` has many `payloads` (plural). this is very important. your has_many's should always be plural because our return value will be a collection of objects, not just one

^^^ this comment goes for all of your associations in your other models as well